zapcore/error_test.go

```diff
@@ -134,10 +174,10 @@ func TestErrorEncoding(t *testing.T) {
 				"errorCauses": []interface{}{
 					map[string]interface{}{
 						"error": "hello: foo; bar",
-						"errorVerbose": "the following errors occurred:\n" +
-							" -  foo\n" +
-							" -  bar\n" +
-							"hello",
+						"errorCauses": []interface{}{
```

Comment:
This is a slight behaviour change, what previously used `errorVerbose` now uses `errorCauses`.